readme-generator/src/components/TemplateCard/Contributors.tsx

```diff
@@ -0,0 +1,56 @@
+'use client'
+
+import { useEffect, useState } from 'react'
+import { FC } from 'react'
+import { AiFillGithub, AiFillTwitterCircle, AiFillLinkedin, AiFillInstagram } from 'react-icons/ai'
+
+interface ContributorProps {
+  contributors: {
+    name: string
+    url: string
+    social: string
+  }[]
+
+}
+
+const Contributors: FC<ContributorProps> = ({
+  contributors
+}) => {
+  const [isMounted, setIsMounted] = useState(false)
```

Comment:
~~Can be a useRef to prevent re-renders, but what is the point of this mount clause? Doesn't seem to have any business logic~~

I see that you want to render this as null if the component is lagging i.e. async data, however I'm not too sure the purpose of this code